src/components/CheckboxLabeled/CheckboxLabeled.jsx

```diff
@@ -83,23 +83,23 @@ const CheckboxLabeled = createClass({
 
 		return (
 			<label
-					className={cx('&', {
-						'&-is-disabled': isDisabled,
-						'&-is-selected': isIndeterminate || isSelected,
-					}, className)}
-					style={style}
+				className={cx('&', {
+					'&-is-disabled': isDisabled,
+					'&-is-selected': isIndeterminate || isSelected,
+				}, className)}
```

Comment:
this is kind of weird, `this.props.className` gets passed to both the root `label` and the `Checkbox`, not sure how to handle this